LoopKit/NotificationSettings.swift

```diff
@@ -15,6 +15,7 @@ public struct NotificationSettings: Codable, Equatable {
         case denied
         case authorized
         case provisional
+        case ephemeral
```

Comment:
It is a straight copy from https://developer.apple.com/documentation/usernotifications/unauthorizationstatus. This `NotificationSettings` structure is a straight copy of `UNAuthorizationStatus`, but is `Codable` (while `UNAuthorizationStatus` cannot be because it is an `NSObject`). This was added a while ago. I'm only updating the enum to get rid of a warning here.